src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1776,6 +1841,18 @@
           short-summary: Space-separated list of services allowed private access to this subnet.
           populator-commands:
             - az network vnet list-endpoint-services
+    examples:
+        - name: Associate a network security group to a subnet
+          text: az network vnet subnet update -n MySubnet --vnet-name MyVNet -g MyResourceGroup --network-security-group MyNSG
+"""
+
+helps['network vnet list-endpoint-services'] = """
+    type: command
+    long-summary: To learn more about service endpoints visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-service-endpoints-configure#azure-cli
+    examples:
+        - name: List the endpoint services available for use in the West US
```

Comment:
Phrasing; use `West US region` and end with a `.`.

Reply:
Fixed in my branch